DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,16 @@ public function getConfigTreeBuilder()
                     ->defaultValue(array())
                     ->prototype('scalar')->end()
                 ->end()
+                ->arrayNode('use_stacktrace_js')
+                    ->info('add StackTrace.js as logging provider')
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('path')
+                            ->isRequired()
```

Comment:
is the path really required? even if I don't use the stacktrace_js feature? or does this only count if "use_stacktrace_js" is "true"?